pkg/apis/servicecatalog/v1alpha1/zz_generated.defaults.go

```diff
@@ -28,5 +28,31 @@ import (
 // Public to allow building arbitrary schemes.
 // All generated defaulters are covering - they call all nested defaulters.
 func RegisterDefaults(scheme *runtime.Scheme) error {
```

Comment:
This file shouldn't be changed, since it is generated.  Put all of this stuff into a `defaults.go` in the same directory, if you plz.